.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "useTabs": false,
+  "tabWidth": 2,
+  "printWidth": 100,
+  "singleQuote": true,
+  "trailingComma": "none"
+}
```

Comment:
if you're adding Prettier, then it's probably better to also add the VSCode setting for this, I don't want Prettier conflict with my other projects but I do want it enable for this project. I have it setup in a work project with this

`.vscode > settings.json`
```json
{
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "editor.formatOnSave": true,
  "editor.formatOnPaste": false
}
```